WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -610,20 +610,11 @@ def for_senior
 
   private def competition_params
     permitted_competition_params = [
-      :use_wca_registration,
-      :external_registration_page,
       :receive_registration_emails,
-      :registration_open,
-      :registration_close,
       :guests_enabled,
       :enable_donations,
```

Comment:
Aren't the two fields above also disabled for confirmed competition?
https://github.com/thewca/worldcubeassociation.org/blob/6c3f6744afa2b87c91b43c6de551f613fc96ac30/WcaOnRails/app/views/competitions/_competition_form.html.erb#L175-L180

Reply:
Oops, right! I've amended the PR.